cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json

```diff
@@ -185,7 +187,17 @@
           "description": "The amount of memory in GB to allocate for a master node",
           "widget-attributes": {
             "default": 4096,
-            "min": 256,
+            "min": 1024,
+            "size": "small"
+          }
+        },
+        {
+          "widget-type": "textbox",
```

Comment:
should we use memory-textbox (it seems to be wrongly named - should just be numeric-textbox)?

Reply:
I see.

Reply:
I think the arrows may be ok, since you can manually enter stuff. But that unit conversion part is strange. It seems like that widget would only work for one scenario. It should probably be improved to include unit conversion optionally, right? Can you create a jira?

Reply:
created https://issues.cask.co/browse/CDAP-14234